src/clj_nakadi_java/core.clj

```diff
@@ -47,37 +48,35 @@
     (typeLiteral [_] TypeLiterals/OF_STRING)))
 
 
-(defn set-param
-  "Sets properties of a StreamConfiguration instance.
-  Takes the value of key from config-map and uses the supplied
-  method to update the instance of StreamConfiguration.
-  Parameters are optional by default."
-  ([StreamConfiguration, config-map, key, method, required]
-   (if-let [value (key config-map)]
-     (method StreamConfiguration value)
-     (do
-       (if required
-         (assert (key config-map) (format "Stream configuration must include %s" key))
-         StreamConfiguration))))
+(defn method-exists? [method]
+  (first (filter #(= (str (:name %)) method) (:members (r/reflect StreamConfiguration)))))
 
-  ([StreamConfiguration, config-map, key, method]
-   (set-param StreamConfiguration config-map key method false)))
+
+(defn ->camel-case [kebab]
+  (let [a (clojure.string/split (name kebab) #"\-")]
```

Comment:
Please use self-descriptive variable name instead of just `a`. `name-parts`, perhaps?

Reply:
I followed your suggestion.